schemas/dab.draft.schema.json

```diff
@@ -395,6 +388,19 @@
               }
             }
           }
+        },
+        "health": {
```

Comment:
how is this `health` different than what is defined on line 38 above? Both are under runtime section right?